tests/Doctrine/Tests/DBAL/Functional/ConnectionTest.php

```diff
@@ -104,7 +104,7 @@ public function testTransactionNestingBehaviorWithSavepoints() : void
             } catch (ConnectionException $e) {
                 self::assertTrue($this->connection->getNestTransactionsWithSavepoints());
             }
-            self::assertTrue($this->connection->commit()); // should not throw exception
+            $this->connection->commit(); // should not throw exception
```

Comment:
is necessary this comment on this line?

Reply:
That's the original code. I think there is no reason to drop the comment in this PR.

Reply:
ok 👍